pkg/template/cluster/cluster.go

```diff
@@ -47,7 +47,7 @@ func NewClusterCRs(config Config) (*apiv1alpha2.Cluster, *infrastructurev1alpha2
 		return nil, nil, nil, nil, microerror.Mask(err)
 	}
 
-	controlPlaneID := key.GenerateID()
+	controlPlaneID := "cp-" + key.GenerateID()
```

Comment:
I am not sure this is a good idea as we work with IDs that are always 5 chars and only letters and numbers, that could break things

Reply:
+1 the reason why we have 5 character IDs is that AWS has strict resource name limits and we use the ID in a lot of places.

Reply:
All right. It was worth a try. ;-)